api/envoy/config/filter/http/compressor/gzip/v2/gzip.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.compressor.gzip.v2;
+
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/migrate.proto";
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.filter.http.compressor.gzip.v2";
+option java_outer_classname = "GzipProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package =
+    "envoy.extensions.filters.http.compressor.gzip.v3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Gzip]
+// [#extension: envoy.filters.http.compressor.gzip]
+
+// [#next-free-field: 10]
+message Gzip {
+  enum CompressionStrategy {
+    DEFAULT = 0;
```

Comment:
Can you comment on what the DEFAULTs are here and below? These should be fixed in the API.